docs/src/content/docs/de/guides/customization.mdx

```diff
@@ -77,21 +82,23 @@ Du kannst verschiedene Versionen deines Logos im Light- und Dark-Mode anzeigen.
 
 2. Füge den Pfad zu deiner Logovarianten als die Optionen `light` und `dark` anstelle von `src` in `astro.config.mjs` ein:
 
-   ```js
+   ```diff lang="js"
    starlight({
      title: 'Dokumentation mit meinem Logo',
      logo: {
-       light: './src/assets/light-logo.svg',
-       dark: './src/assets/dark-logo.svg',
+   +    light: './src/assets/light-logo.svg',
+   +    dark: './src/assets/dark-logo.svg',
      },
    }),
    ```
 
+</Steps>
+
 ## Sitemap aktivieren
 
 Starlight hat eine eingebaute Unterstützung für die Erstellung einer Sitemap. Aktiviere die Sitemap-Generierung, indem du deine URL als `site` in `astro.config.mjs` angibst:
 
-```js
+```js {4}
```

Comment:
I think this section is missing the changes from https://github.com/withastro/starlight/pull/2083/files#diff-596fab03bd0326b4a66f8afc7a97400b00fc458bdf93e1a32bcd72be7b8790f2